stock_orderpoint_uom/views/stock_warehouse_orderpoint_view.xml

```diff
@@ -3,24 +3,14 @@
     <record id="view_warehouse_orderpoint_tree" model="ir.ui.view">
         <field name="name">stock.warehouse.orderpoint.tree</field>
         <field name="model">stock.warehouse.orderpoint</field>
-        <field name="inherit_id" ref="stock.view_warehouse_orderpoint_tree_editable" />
+        <field
+            name="inherit_id"
+            ref="stock.view_warehouse_orderpoint_tree_editable_config"
+        />
         <field name="arch" type="xml">
             <field name="product_uom_name" position="after">
                 <field name="procure_uom_id" groups="uom.group_uom" />
             </field>
         </field>
     </record>
-    <record id="view_warehouse_orderpoint_form" model="ir.ui.view">
-        <field name="name">stock.warehouse.orderpoint.form</field>
-        <field name="model">stock.warehouse.orderpoint</field>
-        <field name="inherit_id" ref="stock.view_warehouse_orderpoint_form" />
-        <field name="arch" type="xml">
-            <field name="product_id" position="after">
-                <field name="procure_uom_id" class="oe_inline" groups="uom.group_uom" />
-            </field>
-            <field name="qty_multiple" position="after">
-                <p class="oe_grey">Quantity Multiple is applied to the base UoM.</p>
-            </field>
-        </field>
-    </record>
```

Comment:
Why removing the form view?

![image](https://github.com/user-attachments/assets/ea13eacd-f715-4e18-acf0-b0b6cbc40606)

I still think it is a good idea to include the procure uom there.